### PR TITLE
Trim stats in interractive summary

### PR DESCRIPTION
**Pull request by emlynjdavies**
quick adding of the 'trim stats' tool into the summary GUI to help with faster data exploring without reloading large files all the time.

### CODE REVIEW

File: pysilcam/silcamgui/interactive_summary.py [hunk 1]
```diff
@@ -600,6 +618,47 @@ def save_data(self):
         self.update_plot(save=True)
 
 
+    def save_trimmed_stats(self):
```
Comment (tankefugl) [inline pysilcam/silcamgui/interactive_summary.py:621]: E303: too many blank lines (2)

File: pysilcam/silcamgui/interactive_summary.py [hunk 2]
```diff
@@ -600,6 +618,47 @@ def save_data(self):
...
+        start_time = self.mid_time - self.av_window / 2
+        end_time = self.mid_time + self.av_window / 2
+
+        if len(self.stats)==0:
```
Comment (tankefugl) [inline pysilcam/silcamgui/interactive_summary.py:625]: E225: missing whitespace around operator

**Review by tankefugl — CHANGES_REQUESTED** [review]
[Lintly](https://github.com/grantmcconnaughey/Lintly) has detected code quality issues in this pull request.

**Review by tankefugl — APPROVED** [review]
No linting violations have been found in this PR.

**Review by bjarnekvae — APPROVED** [review]

**Review by tankefugl — APPROVED** [review]
No linting violations have been found in this PR.

**Review by tankefugl — APPROVED** [review]
No linting violations have been found in this PR.

**Pull request merged**